deploy/crds/sriovnetwork_v1_sriovnetworknodepolicy_crd.yaml

```diff
@@ -39,6 +39,9 @@ spec:
             isRdma:
               description: RDMA mode. Defaults to false.
               type: boolean
+            linkType:
+              description: Link Type for NIC
```

Comment:
It would be nice put the allowed value in the description. Also an `enum` of allowed values can help the API validation.

Reply:
Good idea